### PR TITLE
Github and Google login test

### PR DESCRIPTION
**Pull request by ATHARVA-GAWAS**

### CODE REVIEW

File: website/tests.py [hunk 1]
```diff
@@ -145,3 +151,53 @@ def test_on_hide(self):
...
+                raise ValueError("Unexpected URL name")
+
+        def mock_build_absolute_uri(uri):
+            return "http://example.com" + uri
+
+        with patch("django.urls.reverse", side_effect=mock_reverse):
+            with patch("django.http.HttpRequest.build_absolute_uri", side_effect=mock_build_absolute_uri):
+                callback_url = view.callback_url
+
+        self.assertEqual(callback_url, "http://example.com/accounts/google/login/callback/")
```
Comment (DonnieBLT) [inline website/tests.py:178]: This doesn't seem to be testing anything? What is example.com why are we testing their site?  Also what issue is this solving?

**Review by DonnieBLT — CHANGES_REQUESTED** [review]
See comment

File: website/tests.py (hunk 1 above)
Comment (ATHARVA-GAWAS) [reply website/tests.py:178]: How can i directly use BLT domain??

Comment (ATHARVA-GAWAS) [reply website/tests.py:178]: Need to figure that out

**Pull request closed without merging**